# Feedback.msg
string axis_name
string status
string state
int32 error_id
string timestamp
bool position_reached
